darwin/exporter/formats/yolo.py

```diff
@@ -0,0 +1,90 @@
+from functools import partial
+from pathlib import Path
+from typing import Any, Callable, Dict, Iterable, List, Tuple, Union
+
+from rich.console import Console
+from rich.theme import Theme
+
+from darwin.datatypes import (
+    Annotation,
+    AnnotationFile,
+    ConversionError,
+    VideoAnnotation,
+    YoloAnnotation,
+)
+
+
+def export(annotation_files: Iterable[AnnotationFile], output_dir: Path) -> None:
+    """
+    Exports the given ``AnnotationFile``s into the yolo format inside of the given ``output_dir``.
+
+    Parameters
+    ----------
+    annotation_files : Iterator[dt.AnnotationFile]
+        The ``AnnotationFile``s to be exported.
+    output_dir : Path
+        The folder where the new coco file will be.
+    """
+    console = Console(theme=_console_theme())
+
+    output_dir.mkdir(parents=True, exist_ok=True)
+    for annotation_file in annotation_files:
+        output_file_path = (output_dir / annotation_file.filename).with_suffix(".txt")
+        errors, yolo_annotations = _convert_file(annotation_file)
+
+        with open(output_file_path, "w") as f:
+            for yolo in yolo_annotations:
+                f.write(f"{yolo.annotation_class} {yolo.x} {yolo.y} {yolo.width} {yolo.height}")
```

Comment:
I'm sure this is the right format, but I think we could include something about the format in this module's documentation

Reply:
Done